### ISSUE TITLE
[portal]: make it easier for the users to contact. 

### ISSUE THREAD

**Issue by nisarhassan12** [NeonLaw/codebase#2345]
In a few places on the portal we ask the user to contact us i.e on bussiness and litigation pages but then there is no link to the contact page or a contact form maybe we should link to the contact page from here ? 

![image](https://user-images.githubusercontent.com/46004116/117580813-3bc20c80-b113-11eb-9113-a1ce3658940c.png)

Also the the following is how the contact form looks at the moment which does not seem to cover what to do If one is already a client we should probably addresses that as well:

![image](https://user-images.githubusercontent.com/46004116/117582896-b42dcb00-b11d-11eb-9086-6e1010447868.png)

**Comment by shicholas** [comment]
In production, we have the Zendesk Chat which people can use to contact us. Maybe we can expose a form that creates a Zendesk ticket?

**Comment by nisarhassan12** [comment]
Thanks. Can you tell what a zendesk ticket is and how it can help us here ? 

![image](https://user-images.githubusercontent.com/46004116/117583528-01f80280-b121-11eb-9934-7956d460cfbd.png)

I see the form when I click the help button in the bottom right maybe we can add a button below that opens the zendesk chat just below where it says contact ? 

![image](https://user-images.githubusercontent.com/46004116/117583572-46839e00-b121-11eb-8dae-c52f0a0f70d1.png)

**Comment by shicholas** [comment]
yes that sounds good, Zendesk is like a shared email system where more than me can answer requests
